xc/xc7/fasm2bels/make_routes.py

```diff
@@ -314,9 +325,10 @@ def descend_fixed_route(source_node_pkey, fixed_route):
 
                 yield ')'
 
-            # TCL cannot express 1-length list, so add an additional element
-            # to prevent TCL from collapsing the 1-length list.
-            yield '{} ]'
+            if len(source_nodes) == 1:
+                yield ' }'
+            else:
+                yield '{} ]'
```

Comment:
I think we can change `'{} ]'` to `]` because the 1-length case is handled differently

Reply:
Sure, fixed